codec_specs.md

```diff
@@ -183,6 +183,26 @@ Description: The Matroska video stream will contain a demuxed Elementary Stream
 
 Initialization: none
 
+### V_MPEG4/ISO/AVC
+
+Codec ID: V_MPEG4/ISO/AVC
+
+Codec Name: AVC/H.264
+
+Description: Individual pictures of AVC/H.264 stored as described in [@!ISO.14496-15.2014].
```

Comment:
Is it pictures or fields ? In other words interlaced content should group two fields together ? Does the ISO spec refer to fields or pictures ?

Reply:
> Is it pictures or fields ?

I used "picture" on purpose: AFAIU, in AVC and HEVC specs "pictures" refer to either a frame (progressive content) or to a field (interlaced content) or to 2 fields (progressive coded interlaced aka PPF / Picture-per-field?). Neutral wording.

Note that PPF is more or less classic in MPEG-2 Video and current Matroska spec is not more precise about that. currently "Elementary Stream (ES)" wording is used for MPEG-2 Video, would you prefer this wording?

> In other words interlaced content should group two fields together ?

I kept the wording neutral because AFAIK there is no limitation here: as in other formats (MP4, MXF...) all is possible.

In my opinion we should not limit here the possibilities, and the general specification would apply i.e. the element having the content has a timestamp which is the timestamp of the first stream element, as for other formats e.g. MPEG-2 Video.
If you prefer to limit, please indicate which limitation you want.

> Does the ISO spec refer to fields or pictures?

it uses "elementary stream" and "sample" and "picture" a bit randomly without defining them precisely. "elementary stream" seems to be more used, so we could use this wording.

Reply:
OK for picture if that can represent a field in interlaced content.